### ISSUE TITLE
<C-PageUp> and <C-PageDown> doesn't work

### ISSUE THREAD

**Issue by name-snrl** [zellij-org/zellij#3389]
# Issues with the Zellij UI / behavior / crash

**Basic information**

```fish
name_snrl in ~ 
fish > zellij --version
zellij 0.40.1

name_snrl in ~ 
fish > stty size
41 174

name_snrl in ~ 
fish > uname -av
Linux t440s 6.8.6-zen1 #1-NixOS ZEN SMP PREEMPT_DYNAMIC Tue Jan  1 00:00:00 UTC 1980 x86_64 GNU/Linux
```

## Issue description
`<C-PageUp>` and `<C-PageDown>` doesn't work. These are some of the standard mappings in neovim for navigating to the next/previous tab.

## Other relevant information

**Comment by SecretPocketCat** [comment]
Should be fixed by #3383

**Comment by name-snrl** [comment]
it is, thanks
